test/JDBC/jdbc_schedule

```diff
@@ -149,3 +149,8 @@ ignore#!#load_road
 ignore#!#load_tenk
 ignore#!#load_stud_emp
 ignore#!#load_student
+
+# These tests are running in multidb mode in upgrade and singledb mode in JDBC
+ignore#!#BABEL-4279-vu-prepare
+ignore#!#BABEL-4279-vu-verify
+ignore#!#BABEL-4279-vu-cleanup
```

Comment:
why?

Reply:
These test cases will work with upgrade as they are running in multidb mode.